near-contract-standards/src/non_fungible_token/mod.rs

```diff
@@ -21,6 +18,5 @@ mod utils;
 pub use utils::*;
 
 pub use self::core::NonFungibleToken;
-pub use macros::*;
```

Comment:
Actually, let's deprecate macros instead of removing them, so we don't break the existing code.

Reply:
Good idea, I'm not sure what the adoption rates for that are, but better safe than sorry.